pkg/util/object_test.go

```diff
@@ -0,0 +1,61 @@
+package util
+
+import (
+	"fmt"
+	"reflect"
+	"testing"
+)
+
+type sampleInnerStruct struct {
+	Name   string
+	Number int
+	List   []string
+	Map    map[string]string
+}
+
+type sampleStruct struct {
+	Name         string
+	Inner        sampleInnerStruct
+	Ptr          *sampleInnerStruct
+	MapInMap     map[string]map[string]string
+	ArrayInArray [][]string
+	Array        []string
+}
+
+func TestVisitObjectStringsOnStruct(t *testing.T) {
```

Comment:
@kargakis I was planning to add more test functions (OnStruct, onArray, onMap)....